charts/osm/templates/cleanup-hook.yaml

```diff
@@ -1,28 +0,0 @@
-apiVersion: batch/v1
-kind: Job
-metadata:
-  name: {{ .Release.Name }}-cleanup
-  namespace: {{ include "osm.namespace" . }}
-  labels:
-    {{- include "osm.labels" . | nindent 4 }}
-  annotations:
-    helm.sh/hook: post-delete
-    helm.sh/hook-delete-policy: before-hook-creation,hook-succeeded
-spec:
-  template:
-    metadata:
-      name: {{ .Release.Name }}-cleanup
-      labels:
-        {{- include "osm.labels" . | nindent 8 }}
-    spec:
-      serviceAccountName: {{ .Release.Name }}-cleanup
-      restartPolicy: Never
-      containers:
-        - name: garbage-collector
-          image: bitnami/kubectl
-          imagePullPolicy: IfNotPresent
-          command:
-            - sh
-            - -c
-            - >
-             kubectl delete --ignore-not-found configmap -n '{{ include "osm.namespace" . }}' osm-config;
```

Comment:
This should delete the default mesh config going instead.

Reply:
good catch, I added it back and updated the file to delete the mesh config instead